public/react/components/PagesList.js

```diff
@@ -1,34 +1,34 @@
-import React, { useState, useEffect } from 'react';
-import { Page } from './Page';
-import apiURL from '../api';
-import { Article } from './Article';
-export const PagesList = ({pages, setPages}) => {
+import React, { useState, useEffect } from 'react'
+import { Page } from './Page'
+import apiURL from '../api'
+import { Article } from './Article'
+export const PagesList = ({ pages, setPages }) => {
+  const [articleData, setArticleData] = useState('')
 
-	const [articleData, setArticleData] = useState("");
+  async function viewArticle(slug) {
+    try {
+      const response = await fetch(`${apiURL}/wiki/${slug}`)
+      const articleData = await response.json()
+      setPages([])
```

Comment:
Also here I think we need to `setPages([])` since its "empty state" is an empty array not an empty string